src/wp-admin/includes/plugin.php

```diff
@@ -63,6 +63,7 @@
  *     @type bool   $Network     Whether the plugin can only be activated network-wide.
  *     @type string $RequiresWP  Minimum required version of WordPress.
  *     @type string $RequiresPHP Minimum required version of PHP.
+ *     @type string $UpdateID    ID of the plugin for update purposes, should be a URL.
```

Comment:
Shouldn't this be `$UpdateURI`?

Reply:
Yes, I used `UpdateID` while writing it, and missed quite a few references :)